packages/core/parcel/src/cli.js

```diff
@@ -88,6 +88,7 @@ applyOptions(serve, commonOptions);
 let watch = program
   .command('watch [input...]')
   .description('starts the bundler in watch mode')
+  .option('--watch-for-stdin', 'exit when stdin closes')
```

Comment:
Should we also add this to the `serve` command?

Reply:
I do not use `serve` command so I didn't know if this would be useful for anyone, but it can be added.